<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoCalculator</class>
 <widget class="QDialog" name="GeoCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Калькулятор</string>
   </property>
   <widget class="QRadioButton" name="rB_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обратный</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Прямой</string>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_xA">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-20000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_yA">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-20000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_S">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-20000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_degrees">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_min">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sec">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_xB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-20000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_yB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-20000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>20000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_get_result">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>get result</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
